infrastructure-formula/README.md

```diff
@@ -1,15 +1,56 @@
 # Infrastructure Salt states
 
-The states in this directory share the `infrastructure` pillar dictionary to configure virtual machines and virtualization clusters.
+The states in this directory extend other formulas using the `infrastructure` pillar.
 
 The code under this directory is highly opinionated and specific to our infrastructure.
 
 ## Available states
 
-`libvirt.domains`
+### Libvirt:
+
+`infrastructure.libvirt.domains`
 
 Writes virtual machine domain definitions.
 
-`suse_ha.resources`
+### SUSE HA:
+
+`infrastructure.suse_ha.resources`
 
 Configures virtual machine cluster resources.
+
+### Salt:
+
+The Salt states depend on the [Salt formula](https://github.com/saltstack-formulas/salt-formula) and a modified version of the [Podman formula](https://github.com/lkubb/salt-podman-formula). The latter is yet to be upstreamed.
+
+`infrastructure.salt.master`
+
+Configure a Salt master.
+
+`infrastructure.salt.syndic`
+
+Configure a Salt Syndic, which includes a Salt master.
+
+`infrastructure.salt.minion`
+
+Configures a Salt Minion.
+
+`infrastructure.salt.proxy_master`
+
+Extends a Salt Master with capabilities to manage proxy minions.
+
+`infrastructure.salt.proxy_networkautomation`
+
+Configures a container host to run Salt Proxy minions.
+
+`infrastructure.salt.minion_networkautomation`
+
+Extends a container host running Salt Proxy minions to run regular Salt Minions.
+This is for managing devices using Salt states not implementing Salt Proxy operation and instead rely on modules on a regular minion to forward requests to an API.
+
+`infrastructure.salt.scriptconfig`
+
+Writes configuration used by Salt related scripts, currently only `salt-keydiff`.
+
+`infrastructure.salt.common`
+
+Writes configuration shared by Salt services, currently only a `logrotate` drop-in.
```

Comment:
`common.sls` is missing from this PR.

Besides that, wouldn't it be better to get the logrotate config into the salt RPMs?

Reply:
Ah, yes, `common` was here before, when I realized the file matches the package shipped one 1:1, so I removed it. Will remove the README reference promptly!